_posts/blog/2018-11-02-el-topo-blindado.md

```diff
@@ -0,0 +1,59 @@
+---
+
+layout: post
+title: "El Topo Blindado: Centro de documentación de las organizaciones politico-militares argentinas"
+modified:
+categories: blog
+excerpt: ""
+tags: []
+author: nourry
+image:
+  feature:
+
+---
+
+## *El Topo Blindado: Centro de documentación de las organizaciones politico-militares argentinas.* 
+**eltopoblindado.com** 
+*Accessed 10 November 2018. </br> Reviewed by Daniel Nourry, University of Texas at Austin*
+
+A significant proportion of the twentieth century would see the people of Argentina, much like many if not all of the peoples of Latin America in the same period, governed by military dictatorships. Archive projects like the Archivo Histórico de la Policía Nacional de Guatemala are an important resource for understanding how these violent and repressive process functioned. As Kirsten Weld, author of *Paper Cadavers* - a book that records the events that led to the discovery and recovery of these important documents - tells us "People study history in order to participate in contemporary politics; we recover the past in order to look to the future. As such, documents, archives, and historical knowledge are more than just the building blocks of politics- they are themselves sites of contemporary political struggle" (3). In other words, the recovery, maintenance and promotion of archival records and sources is in itself a political act: a way of participating in the struggle. 
```

Comment:
The long discussion of the AHPN here is distracting. I would cut it - you can quote Weld without describing that project in depth.